x-pack/plugins/security_solution/public/detections/components/value_lists_management_modal/table_helpers.tsx

```diff
@@ -29,6 +29,11 @@ export const buildColumns = (
     name: i18n.COLUMN_FILE_NAME,
     truncateText: true,
   },
+  {
+    field: 'type',
+    name: i18n.COLUMN_TYPE,
+    width: '20%',
```

Comment:
It's probably worth adding `truncateText: true` here as well. I was also able to drop the width to 15% and still have room for all the current types, that'll give a little more room for the filename:

<kbd><img width="758" alt="Detections_-_Kibana" src="https://user-images.githubusercontent.com/657252/94082500-b3cb0480-fdc6-11ea-9559-c2fbcc234654.png"></kbd>

One other thing worth discussing with this column: we display the humanized values in the dropdown, but the table then shows the actual values, which is potentially confusing to the user. We may want to decorate these values similarly to the date column.

Reply:
👍 switched over to prettier words